docs/docs-content/clusters/edge/local-ui/cluster-management/local-to-central.md

```diff
@@ -0,0 +1,189 @@
+  cluster.
+
+- Your Palette environment has cluster profiles whose latest version exactly match all layers of the local cluster.
+  Refer to [Accept Pairing Request from Palette](#accept-pairing-request-from-palette) for details and examples.
+
+### Procedure
+
+#### Send Pairing Request from Local Cluster
+
+1. Log in to [Local UI](../host-management/access-console.md).
+
+2. On the left **Main Menu**, click **Settings**.
+
+3. On the **Settings** tab, click on the radio button that says **Centrally Managed**. You will be prompted to enter
+   information required for pairing your cluster with Palette.
+
+4. Enter the URL for the Palette API endpoint. If you are using Palette SaaS, the endpoint is `api.spectrocloud.com`.
+
+5. Enter your registration token in the **Registration token** field.
+
+6. Click **Pair** to send the pairing request. A pairing ID will be displayed on the Local UI screen once a pairing
+   request has been sent. You may use the ID to identify this pairing request in a later step.
+
+#### Accept Pairing Request from Palette
+
+7. Log in to [Palette](https://console.palette.com).
+
+8. From the left **Main Menu**, click **Clusters**.
+
+9. At the top of the page, a box will be displayed prompting you to review pairing requests that Palette has received.
+   Click **Pair** to start reviewing the requests.
+
+10. Identify the pairing request corresponding to your cluster by its pairing ID. Click **Pair**.
+
+11. Palette will warn you that once paired, the cluster can only be managed centrally and that local cluster operations
+    will no longer be permitted. Check the box that says **I understand** and click **Confirm**. This will take you to
+    the **Pair Cluster** wizard.
+
+12. You may change the cluster name as well as the description and tags for your cluster. The fields will be
+    pre-populated with their existing values. Once you are satisfied with your changes, click **Next step**.
+
+13. Match the profile used by your local cluster by selecting profiles in your Palette environment. To make a successful
+    match, ensure that every pack in the profile used by your cluster is matched with the same pack in the exact same
+    version. You can only choose the latest version of a profile in Palette and must make sure that the latest version
+    of the profiles you use can match the cluster.
+
+    In the following example, the second and third columns are valid matches because every layer in the original cluster
+    has a matching layer in the matched profile. **Valid match 2** is valid even though it only has one full profile
+    instead of an infra and an add-on profile, because all the layers are exactly the same.
+
+    The fourth column shows an example of an invalid match because the **Add-on** profile has a pack that is in the
+    wrong version. The fifth column is also invalid because it has an additional pack that is not found in the original
+    cluster's profile.
```

Comment:
```suggestion
    wrong version. The fifth column is also invalid because it has an additional pack that is not found in the local cluster's cluster profile.
```
Just a minor clarification